fedora/templates/csv/packages_removed.csv

```diff
@@ -5,3 +5,4 @@ prelink
 setroubleshoot
 xorg-x11-server-common
 sssd
+systemd
```

Comment:
@jan-cerny are you sure you want to remove `systemd` from the system?

Reply:
@dahaic Yes, I am. :smiley_cat: 

It's consistent with what we do in RHEL 7.  have to add systemd to a list of
removed packages because debug-shell is provided by systemd package. If we forget about it, the respective OVAL definition will be removed from the final datastream by our build system. Check how does the OVAL template for service_.*disabled_rules look like.

Reply:
Ah, makes sense! :)